extension/docs/uddsketch.md

```diff
@@ -242,7 +256,7 @@ This will combine multiple already constructed UddSketches, they must have the s
 | `uddsketch` | `UddSketch` | A UddSketch object which may be passed to other UddSketch APIs. |
 <br>
 
-### Sample Usages <a id="uddsketch-examples"></a>
+### Sample Usages <a id="uddsketch-summary-examples"></a>
 For this examples assume we have a table 'samples' with a column 'data' holding `DOUBLE PRECISION` values, and an 'id' column that holds the what series the data belongs to, we can create a view to get the UddSketches for each `id` using the [point form](#uddsketch-point) like so:
```

Comment:
seems like "this examples" is all over the place, may want to do a find/replace.